src/modules/assets/views/ViewAssetsList.vue

```diff
@@ -0,0 +1,236 @@
+<script>
+    //used for testing
+    import axios from 'axios'
+    export default{
+    components: [
+    ],
+    data() {
+        return {
+            assets: []
+        };
+    },
+    components: {  },
+    async created(){
+      //this block is used for testing
+      try{
+        const response = await axios.get(`https://6319958e8e51a64d2be7568b.mockapi.io/assetslist`);
+        this.assets = response.data;
+      }catch(e){
+        console.error(e);
```

Comment:
Use toast notifications instead of the console.log

Reply:
- removed conaole.log() 
- the toast notification is being implemented on a seperate card and will be used after this card is merged